float64               abratio      
int8                  detectcolor  
bool                  detect_done 
int8                  reportgoal
geometry_msgs/Pose2D   odometrie_1 
geometry_msgs/Pose2D   odometrie_2

